services/todo-legacy/README.md

```diff
@@ -0,0 +1,39 @@
+# Todo Application with Legacy Juggler
+
+## Setup
+
+### Basic Usage
+To use this example:
+```
+# Install typescript and ts-node
+npm i -g typescript ts-node
```

Comment:
It should also be in the in the "Installation" section as per http://loopback.io/doc/en/contrib/README-guidelines.html#installation